spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/ErrorMessageSubmit.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spark.commons.errorhandling
+
+import za.co.absa.spark.commons.errorhandling.types._
+
+trait ErrorMessageSubmit {
```

Comment:
This trait needs `errCol` - I can see that there is `errCols` already, but if I understand the intentions here, the idea is to have 2 attributes - one for a single column name and the other for potentially multiple columns

Reply:
(my comment was related to a very old version of this file, thus is not relevant any more)